test/http_body_test.dart

```diff
@@ -116,7 +116,8 @@ void _testHttpServerRequestBody() {
       }
       body.request.response.close();
     }, onError: (error) {
```

Comment:
```suggestion
    }, onError: (Object error) {
```

And then remove the cast below.